ktor-server/ktor-server-plugins/ktor-server-htmx/common/src/io/ktor/server/htmx/HxAttributes.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.htmx
+
+import io.ktor.htmx.HxAttributeKeys
+import kotlinx.html.HtmlTagMarker
+import kotlinx.html.impl.DelegatingMap
+import kotlin.jvm.JvmInline
+
+@ExperimentalHtmxApi
+public val DelegatingMap.hx: HxAttributes get() = HxAttributes(this)
+
+@ExperimentalHtmxApi
+public inline fun DelegatingMap.hx(block: HxAttributes.() -> Unit) {
+    hx.block()
+}
+
+@ExperimentalHtmxApi
+@HtmlTagMarker
+public class HxAttributes(override val map: DelegatingMap) : StringMapDelegate {
+    public var get: String? by HxAttributeKeys.Get
+    public var post: String? by HxAttributeKeys.Post
+    public var pushUrl: String? by HxAttributeKeys.PushUrl
+    public var select: String? by HxAttributeKeys.Select
+    public var selectOob: String? by HxAttributeKeys.SelectOob
+    public var swap: String? by HxAttributeKeys.Swap
```

Comment:
We've had type-safe implementations for some of these fields. Will it be part of some future PR?